api/docker-compose.yaml

```diff
@@ -2,7 +2,7 @@ version: "3.4"
 x-api-defaults: &api-defaults
   build:
     context: ./src/dev/server
-  image: allofustest/workbench-dev-api:local
+  image: allofustest/workbench-dev-api:local-1
```

Comment:
Does this image exist on Dockerhub?  I don't have access to it.

Reply:
It doesn't - I just added a comment above this line in my other PR which you should now be able to see here. Please take a read and let me know if this addresses your question.